check.rb

```diff
@@ -16,8 +16,9 @@
 begin
   require 'openssl'
 rescue LoadError
-  abort "Oh no! Your Ruby doesn't have OpenSSL, so it can't connect to #{host}. " \
-    "You'll need to recompile or reinstall Ruby with OpenSSL support and try again."
+  puts "Oh no! Your Ruby doesn't have OpenSSL, so it can't connect to #{host}.",
+       "You'll need to recompile or reinstall Ruby with OpenSSL support and try again."
+  exit_code = 1
```

Comment:
`exit_code` is never referenced anywhere else. You should be able to just replace this with `exit 1`.

```suggestion
  exit 1
```

Reply:
Fixed.